src/Path.php

```diff
@@ -89,6 +91,11 @@ public static function canonicalize($path)
             return self::$buffer[$path];
         }
 
+        // Replace "~" with user's home directory.
+        if ('~' === $path[0]) {
+            $path = static::getHomeDirectory() . substr($path, 1);
```

Comment:
please remove the spaces around the `.` here and everywhere else

I'll create a PHPCS-fixer soon for stuff like that :)